viite-UI/src/model/ProjectCollection.js

```diff
@@ -332,6 +332,29 @@
       var coordinates = applicationModel.getUserGeoLocation();
       var roadAddressProjectForm = $('#roadAddressProjectForm');
       var endDistance = $('#endDistance')[0];
+      const hasDevRights = _.includes(startupParameters.roles, 'dev');
+
+      const startAddrMValue = roadAddressProjectForm.find('#addrStart').length && roadAddressProjectForm.find('#addrStart')[0].value ? Number(roadAddressProjectForm.find('#addrStart')[0].value) : null;
```

Comment:
(nil: tahtoo sarakesijoittelun =- ja &&- ja vastaavista kohdista)

Reply:
Ois helpompaa ainakin mun silmille ja aivoille, jos näitä myös jakaisi parille riville yhden pitkän rimpsun sijaan